java/tools/src/java/org/apache/orc/tools/ScanData.java

```diff
@@ -40,41 +40,168 @@
 
   static CommandLine parseCommandLine(String[] args) throws ParseException {
     Options options = new Options()
-        .addOption("help", "h", false, "Provide help");
-    return new GnuParser().parse(options, args);
+        .addOption("s", "schema", false, "Print schema")
+        .addOption("h", "help", false, "Provide help");
+    return new DefaultParser().parse(options, args);
```

Comment:
Oh, are we moving from `GnuParser` to `DefaultParser`? I'm just wondering because we have some more `GnuParser` code on the `master` branch.
```
$ git grep DefaultParser | wc -l
      12
$ git grep GnuParser | wc -l
      10
```

Reply:
Intellij complained that GnuParser is deprecated and it has been replaced with DefaultParser.